libqtile/window.py

```diff
@@ -239,7 +239,7 @@ def __init__(self, window, qtile):
 
     x = property(fset=_geometry_setter("x"), fget=_geometry_getter("x"))
     y = property(fset=_geometry_setter("y"), fget=_geometry_getter("y"))
-    z = property(fset=_geometry_setter("z"), fget=_geometry_getter("z"))
+    z = property(fget=_geometry_getter("z"))
```

Comment:
I think you can fold this commit into the one that adds it

Reply:
Done.